packages/console/src/contexts/TenantsProvider.tsx

```diff
@@ -25,7 +25,17 @@ export enum GlobalAnonymousRoute {
   SocialDemoCallback = '/social-demo-callback',
 }
 
-const anonymousRoutes: Readonly<string[]> = Object.freeze(Object.values(GlobalAnonymousRoute));
+/**
+ * The routes start with a tenant ID but need tenant access.
+ */
+export enum GlobalRoute {
+  CheckoutSuccessCallback = '/checkout-success-callback',
```

Comment:
Why does this route start with a slash, when you have to drop the slash in every use case?

Reply:
We use it in 2 places.
when this route is under the `/:tenantId/` path, it can not be a absolute path.
But when it's not under the `/:tenantId/` path, we can use it directly.

> Absolute route path "/checkout-success-callback" nested under path "/:tenantId/" is not valid. An absolute child route path must start with the combined path of all its parent routes.